swagger.yaml

```diff
@@ -23,6 +23,16 @@ paths:
           description: "area types returned successfully"
           schema:
             $ref: '#/definitions/GetAreaTypesResponse'
+        401:
+          $ref: '#/responses/Unauthorised'
+        404:
+          description: "Cantabular Blob not found"
```

Comment:
we're supposed to keep the word "Blob" out of the public API / swagger spec according to e.g. @eldeal

Reply:
....Really could of done with knowing that sooner :)

Reply:
I will probably need to rename the query param to `cantabular_dataset` as well then

Reply:
I will rename canbaular_blob to canbaular_dataset (except for the hello endpoint, which will be removed at some point)

Reply:
yeh i think dataset would be preferable please :)